.deployment.config.json

```diff
@@ -40,6 +51,20 @@
         }
       }
     },
+    {
+      "id": "deploy-atomic-to-s3-version",
+      "s3": {
+        "bucket": "{terraform.infra.infra.bucket_binaries}",
+        "directory": "proda/StaticCDN/atomic/v$[PRERELEASE]",
+        "source": "packages/headless/dist/atomic",
```

Comment:
What would the full url look like to pull in atomic from the CDN?

Reply:
It would look like:
* https://static.cloud.coveo.com/atomic/latest/atomic.esm.js
* https://static.cloud.coveo.com/atomic/v0.11.0/atomic.esm.js